hardware/raspberrypi/bootmodes/nvme.md

```diff
@@ -54,14 +54,13 @@ File read: pieeprom.bin
 Second stage boot server done
 ```
 
-### Firmware and Kernel
+### Firmware and kernel
 
-At the time of writing you need to use pre-release versions of the VideoCore firmware and kernel and these need to be on the NVMe disk. When this software is officially released you will just be able to write the software to the NVMe disk directly with the rpi-imager app. Until then the easiest way to do this is to...
-
-* Boot the board with a blank SSD plugged into the PCIe slot
-* Use [piclone](https://github.com/raspberrypi-ui/piclone) to copy the image to the NVMe disk. When using piclone choose the option to use "New Partition ids"
-* Update the software on the NVMe disk with the latest pre-release software using [rpi-update](../../../raspbian/applications/rpi-update.md).
+You must load pre-release versions of the VideoCore firmware and kernel to the NVMe disk. When this software is officially released you can simply write the software to the NVMe disk directly with the rpi-imager app: until then the easiest way to perform the update is:
```

Comment:
The colon at the end of the line should probably stay as-is, but the colon before that should become a semi-colon?

Reply:
Fixed.